src/rules/converters/tests/file-name-casing.test.ts

```diff
@@ -0,0 +1,106 @@
+import { convertFileNameCasing } from "../file-name-casing";
+
+describe(convertFileNameCasing, () => {
+    test("conversion without parameter", () => {
+        const result = convertFileNameCasing({
+            ruleArguments: [],
+        });
+
+        expect(result).toEqual({
+            rules: [
+                {
+                    ruleName: "unicorn/filename-case",
+                },
+            ],
+            plugins: ["unicorn"],
+        });
+    });
+
+    test("conversion without string value", () => {
+        const result = convertFileNameCasing({
+            ruleArguments: [true, "camel-case"],
+        });
+
+        expect(result).toEqual({
+            rules: [
+                {
+                    ruleName: "unicorn/filename-case",
+                    ruleArguments: [
+                        {
+                            cases: {
+                                camelCase: true,
+                            },
+                        },
+                    ],
+                },
+            ],
+            plugins: ["unicorn"],
+        });
+    });
+
+    test("conversion with object for filetypes", () => {
+        const result = convertFileNameCasing({
+            ruleArguments: [true, { ".ts": "ignore", ".tsx": "pascal-case", ".js": "snake-case" }],
```

Comment:
Aha, a behavior change! TSLint can specify file name casing per file but ESLint just lets all files have any case. There should be another `notices` member here for the difference.

Reply:
I added a notice: `ESLint (Unicorn Plugin) does not support file name casing by file type, so all previously configured casings are now allowed.`